src/software/handheld_controller/primitive_generator.cpp

```diff
@@ -0,0 +1,56 @@
+#include "software/handheld_controller/primitive_generator.h"
+
+#include "software/ai/primitive/chip_primitive.h"
+#include "software/ai/primitive/direct_velocity_primitive.h"
+#include "software/ai/primitive/kick_primitive.h"
+
+PrimitiveGenerator::PrimitiveGenerator(
```

Comment:
Could we rename to `ControllerPrimitiveGenerator`? Wasn't sure what this was at first.

Reply:
renamed as suggested